src/Apps/Artwork/Utils/createCollectUrl.tsx

```diff
@@ -0,0 +1,46 @@
+import { data as sd } from "sharify"
+
+export const createCollectUrl = (
+  minCents,
+  maxCents,
+  category,
+  widthCm,
+  heightCm,
+  artistId
+) => {
+  const categoryHref = "/collect/" + category.toLowerCase()
+  const acquirableHref =
+    "?page=1&sort=-decayed_merch&acquireable=true&offerable=true&inquireable_only=true"
```

Comment:
Why do we need `inquirable_only=true`? Is `acquirable=true` too broad?

Reply:
I'm not sure exactly. I was going off of the example url in the jira ticket

Reply:
`acquireable=true` will return all artworks that have "Buy now" enabled. `inquireable_only=true` will return all artworks that are inquireable (but not "Buy now"-able).

With `offerable`, `acquireable`, and `inquireable_only`, we'll capture these three checkboxes, which is what we want (since it maps to what we pull in vortex).

![image](https://user-images.githubusercontent.com/2081340/55993789-f73abd00-5c7d-11e9-984c-b986d7ae1121.png)

Reply:
Ahhh I see. My brain interpreted the filters as doing a set intersection rather than a set union. 👍